src/app/components/overlay/subscribe/subscribe.component.html

```diff
@@ -76,14 +76,17 @@
           </section>
 
           <footer class="modal-card-foot">
-            <div class="buttons">
+
+            By signing up you accept our <a href="/assets/files/privacy.txt">Privacy Policy</a>.
```

Comment:
Mag ik een suggestie doen? Lees ook hier: [small semantic HTML meaning](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/small#:~:text=The%20HTML%20element%20represents,from%20small%20to%20x%2Dsmall%20.)

```suggestion
            <small>By signing up you accept our <a href="/assets/files/privacy.txt">Privacy Policy</a>.</small>
```